src/lucky.cr

```diff
@@ -23,4 +23,9 @@ require "./lucky/paginator/components/*"
 module Lucky
   Log              = ::Log.for("lucky")
   ContinuedPipeLog = Log.for("continued_pipe_log")
+
+  # Use Dir.current to return the root folder of your Lucky application.
+  def self.root
```

Comment:
```suggestion
  # Use Dir.current to return the root folder of your Lucky application.
  #
  # In some frameworks there is a method called `root` that returns the root directory of the project.
  # In Crystal there is a built-in method for this: `Dir.current`. This method exists purely to help new users
  # find `Dir.current`. If you call `Lucky.root` it will raise a compile-time error directing you to use `Dir.current`
  def self.root
```